### PR TITLE
Add settings to automation context.

### PR DESCRIPTION
**Pull request by samwho**
## Description

Adds the following items to automation context:

- `settings.url`
- `settings.logo`
- `settings.company`

All taken from the global configuration.

## Launchcontrol

- Add `settings` to the context for automation bindings, allowing you to fetch the installation URL, company logo URL, and company name.

### CODE REVIEW

File: packages/server/src/threads/automation.ts [hunk 1]
```diff
@@ -263,6 +263,13 @@ class Orchestrator {
         this.context.env = await sdkUtils.getEnvironmentVariables()
         this.context.user = this.currentUser
 
+        const { config } = await configs.getSettingsConfigDoc()
```
Comment (adrinr) [inline packages/server/src/threads/automation.ts:266]: Will this always exist? If so, should we make the fields not optional on the type?

Comment (samwho) [reply packages/server/src/threads/automation.ts:266]: I'm going by the types entirely, I'm not sure.

Comment (mike12345567) [reply packages/server/src/threads/automation.ts:266]: If the document exists the config should exist on it - I believe the type is correct as well, it is not optional on it. The get will fail and 404 the endpoint if the document doesn't exist, therefore the `{ config }` will always be safe.

Comment (samwho) [reply packages/server/src/threads/automation.ts]: I _think_ I've implemented what you're suggesting, PTAL 🙏

**Review by mike12345567 — APPROVED** [review]
LGTM - thanks for the fix!

**Pull request merged**